<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>581</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 255);
font: 57 20pt &quot;Poppins Medium&quot;;</string>
    </property>
    <property name="text">
     <string>Permulation Combination</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>461</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Enter N and K:</string>
    </property>
    <widget class="QLabel" name="label_n">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>55</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter N: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_k">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>55</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter K: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_n">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_k">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon.png</normaloff>icon.png</iconset>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>461</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Result:</string>
    </property>
    <widget class="QLabel" name="label_a">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>A=</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_b">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>B=</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_a">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_b">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
